docs/modules/lists/pages/continuation.adoc

```diff
@@ -16,17 +16,23 @@ include::example$complex.adoc[tag=indent]
 include::example$complex.adoc[tag=indent]
 ====
 
-TIP: When items contain more than one line of text, leave a blank line before the next item to make the list easier to read.
+TIP: When list items contain more than one line of text, leave a blank line between items to make the list easier to read while working in the code.
+
+Blank lines between items in an ordered list do not reset numbering to 1 as long as each adjacent line starts with a period. 
+Blank lines do reset numbering to 1 when something, such as a figure or a supporting paragraph, comes between two lines that start with a period.
+To keep numbering continuous in those cases—for example, if you're documenting steps in a procedure—put a `{plus}` symbol in the otherwise blank line. A `{plus}` used in this way is called a list continuation. For details on continuous numbering in an ordered list, see the https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/[Ordered Lists^] section of this documentation. 
```

Comment:
```suggestion
To keep numbering continuous in those cases—for example, if you're documenting steps in a procedure—put a `{plus}` symbol in the otherwise blank line. A `{plus}` used in this way is called a list continuation. For details on continuous numbering in an ordered list, see xref:ordered.adoc[].
```

Hi @MarciaRieferJohnston First off, thanks for helping improve our docs! I appreciate the time you're taking from other things I'm sure you could be doing.

In the suggestion above, I've replaced the link (`https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/[Ordered Lists^]`) with an xref macro. When linking between pages in docs.asciidoctor.org (no matter what project the page belongs to) we use the xref macro and Antora page ID structure for the target (`xref:page-id.adoc[]`). 

A page ID is "a unique and reliable set of identifying coordinates, organized in a standard sequence, that are constructed from the properties assigned to each page’s source file". :grimacing: I hope I didn't just send you running into the night. If you're still with me :crossed_fingers:, you can learn about page IDs here: https://docs.antora.org/antora/3.0/page/page-id/ and see more examples here: https://docs.antora.org/antora/3.0/page/module-and-page-xrefs/ if you're interested.

Long story short, all the pages you've edited in this MR are in the same [module](https://docs.antora.org/antora/3.0/module-directories/) (lists) so to create links between them, just specify the page's file name with .adoc in the xref target. For example, `xref:ordered.adoc[]` to link to the Ordered Lists page, or `xref:unordered.adoc[]` to link to the Unordered Lists page.

I also removed the link text in the suggestion. Antora will fill in the title of the page being referenced automatically when the site is generated.